geode-core/src/distributedTest/java/org/apache/geode/internal/cache/persistence/PersistentRecoveryOrderDUnitTest.java

```diff
@@ -1326,6 +1326,10 @@ public void testRecoverAfterConflict() {
     vm1.invoke(() -> getCache().close());
 
     vm0.invoke(() -> {
+      await().untilAsserted(
```

Comment:
It might be better to just change this test to test the more normal use case of restarting the member that had the ConflictingPersistentDataException. I think you could do that in this test by closing the cache in vm0 when you catch ConflictingPersistentDataException. Then the test would close vm1 and then when vm0 calls createReplicationRegion it will be with a new cache. I don't think we need to test that a member that saw this exception can later on create the region using the same cache.